Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs

```diff
@@ -35,14 +35,12 @@ public static string MixedRealityToolkit_AbsoluteFolderPath
             {
                 if (MixedRealityToolkitFiles.AreFoldersAvailable)
                 {
-#if UNITY_EDITOR
-                    if (MixedRealityToolkitFiles.MRTKDirectories.Count() > 1)
+                    if (Application.isEditor && MixedRealityToolkitFiles.MRTKDirectories.Count() > 1)
```

Comment:
Why do we want to run this if not in editor?

Reply:
I'm not sure what you mean; it's checking if we're in the editor (it's also in an editor-only assembly, so this check is likely unnecessary).